WordPressKit/EditorSettings.swift

```diff
@@ -22,6 +32,8 @@ extension EditorSettings {
     }
 
     private init(with remote: RemoteEditorSettings) {
-        self = EditorSettings(rawValue: remote.editorMobile) ?? .default
```

Comment:
Considering this is only being called from another initializer that `throws`, I wonder if it'd be cleaner to avoid those `default` values and just throw an error if the JSON has unexpected values. Then the app can decide what's the default/fallback value if it can't read one

Reply:
Sounds good. I was unsure about those defaults myself. It's something I was considering asking 🙂